.gradient/prepare-datasets.sh

```diff
@@ -33,24 +11,17 @@ then
 fi
 
 echo "Starting preparation of datasets"
-# symlink exe_cache files
-exe_cache_source_dir="${PUBLIC_DATASET_DIR}/poplar-executables-pytorch-3-1-1"
-symlink-public-resources "${exe_cache_source_dir}" $POPLAR_EXECUTABLE_CACHE_DIR
-# Symlink squad
-symlink-public-resources "${PUBLIC_DATASET_DIR}/squad" "${HF_DATASETS_CACHE}/squad"
-# Symlink OGB Wiki dataset and checkpoint
-symlink-public-resources "${PUBLIC_DATASET_DIR}/ogbl_wikikg2_custom" "${DATASET_DIR}/ogbl_wikikg2_custom"
-
-# symlink local dataset used by vit-model-training notebook
-symlink-public-resources "${PUBLIC_DATASET_DIR}/chest-xray-nihcc-3" "${DATASET_DIR}/chest-xray-nihcc-3"
+cd "$(dirname "$0")"
+python -u ./symlink_datasets_and_caches.py
```

Comment:
Do we need to check the return-code of this script?

Also, if an interpreter line is added to the top of the called script, then it can be invoked directly as:

```bash
./symlink_datasets_and_caches.py
```

Reply:
I don't think we need to check the return code. If something fails in the python script, everything else should still continue. It just means that on the Paperspace platform, the poplar executable caches may not be available and so the user would have to compile from scratch. 

Regarding the interpreter line at the top of the python script - from testing, we need to add the `-u` flag to make the output appear without lag on the Paperspace platform, and I couldn't find a way to do that without invoking `python3 -u ...`, so I've kept it that way for now. If there's a way to do that while still invoking directly then I can make that change.

Reply:
There's some interesting discussion on this at https://unix.stackexchange.com/questions/399690/multiple-arguments-in-shebang - generally, you can only supply up to two arguments via the interpreter/shebang line... but the coreutils version shipped with Ubuntu 19.04 and above (so, note, not 18.04) would let you do:

```bash
#! /usr/bin/env -S python3 -u
```

... at the start of the script to allow it to be executed directly and still inherit the `-u` option!

Reply:
It's strange I get a permission error when I try this on Paperspace:
`2023-03-17T10:26:41.818297809Z stderr F /notebooks/.gradient/prepare-datasets.sh: line 13: ./symlink_datasets_and_caches.py: Permission denied`

Reply:
Updated file permissions, test works now.